index.html

```diff
@@ -49,5 +49,199 @@ <h2 class="job">I’am a Software Developer</h2>
         </li>
       </ul>
     </section>
+    <div class="react-work">
+      <img class="title-text" src="image/Title.svg" alt="" />
+      <img class="work" src="image/react.svg" alt="" />
+      <img class="multi" src="image/multi.svg" alt="" />
+      <img class="pharagraph" src="image/pharagraph.svg" alt="" />
+      <ul class="work-choise">
+        <li class="choise">
+          <a href="#"><img src="image/Tag.svg" alt="" /></a>
+        </li>
+        <li class="choise">
+          <a href=""><img src="image/html.svg" alt="" /></a>
+        </li>
+        <li class="choise">
+          <a href=""><img src="image/bootstrap.svg" alt="" /></a>
+        </li>
+        <li class="choise">
+          <a href=""><img src="image/ruby.svg" alt="" /></a>
+        </li>
+      </ul>
+      <form action="index.html">
+        <button type="submit" class="button1">See Project</button>
+      </form>
+    </div>
+    <div class="wrapper">
+      <div class="profesional-react1">
+        <img class="prof1" src="image/pro.svg" alt="" />
+        <img class="text1" src="image/Supporting_ext.svg" alt="" />
+        <ul class="project-type">
+          <li>
+            <a href=""
+              ><img class="html-opacity" src="image/html.svg" alt=""
+            /></a>
+          </li>
+          <li>
+            <a href=""
+              ><img class="html-opacity" src="image/bootstrap.svg" alt=""
+            /></a>
+          </li>
+          <li>
+            <a href=""
+              ><img class="html-opacity" src="image/ruby.svg" alt=""
+            /></a>
+          </li>
+        </ul>
+        <form action="index.html">
+          <button type="submit" class="projects-button">See Project</button>
+        </form>
+      </div>
```

Comment:
## Changes required
⚠️ You did a great job using a form and a button element. However, according to project requirements, the button (e.g., "See project") is a link `a` or `button` tag, and follow the button styling: Active, Hover and Pressed. Please, kindly change it.